src/config.ts

```diff
@@ -1,14 +1,39 @@
 import { workspace } from 'vscode'
+import type { WorkspaceConfiguration, WorkspaceFolder } from 'vscode'
 export const extensionId = 'zxch3n.vitest-explorer'
 
-export function getConfig() {
-  const config = workspace.getConfiguration('vitest')
+export function getConfigValue<T>(
+  rootConfig: WorkspaceConfiguration,
+  folderConfig: WorkspaceConfiguration,
+  key: string,
+  defaultValue: T,
+): T {
+  return folderConfig.has(key)
```

Comment:
Can this be simplified as `folderConfig.get(key) ?? rootConfig.get(key) ?? defaultValue`